docs/source/quick_start.rst

```diff
@@ -3,23 +3,432 @@ Quick start
 Lochness provides a single command line tool (daemon) to periodically poll
 and download data from various web services into a local directory. Out of
 the box there is support for pulling data from a multitude of 
-`data sources <data_sources.html>`_ including Beiwe, XNAT, REDCap, 
-Dropbox, Box, Mediaflux, Daris, RPMS, external hard drives, and more.
+`data sources <data_sources.html>`_ including REDCap, XNAT, Dropbox, Box,
+Mediaflux, RPMS, external hard drives, and more.
 
 
 
 Installation
 ------------
 Just use ``pip`` ::
 
-    pip install lochness
+    $ pip install ampscz-lochness
```

Comment:
All `$` signs should be removed. It prevents people from copying the command and directly executing it in their terminal. As you understand, one must scroll to the front of the line and remove the `$` sign before scrolling to the end again and hitting enter.

Reply:
Are we hosting a package called `ampscz-lochness` in PyPi now?